README.md

```diff
@@ -96,6 +96,8 @@ The full JDBC url should look like this:
 jdbc:mysql://google/<DATABASE_NAME>?cloudSqlInstance=<INSTANCE_CONNECTION_NAME>&socketFactory=com.google.cloud.sql.mysql.SocketFactory&useSSL=false&user=<MYSQL_USER_NAME>&password=<MYSQL_USER_PASSWORD>
 ```
 
+Note: The host portion of the JDBC url is currently unused, and has no effect on the connection process. The SocketFactory will get your instances IP address base on the provided `cloudSqlInstance` arg. 
```

Comment:
url -> URL

Reply:
also, is this comment specific to MySQL? What about PostgresSQL?

Reply:
Done.

Reply:
while we are at it, can we just upper case the url? :D

Reply:
@saturnism - I don't see any that are still lowercase - do you see one in the current version? I was having some issues with Chrome/Github's search function stepping on each other, but I in the page preview they are all uppercase'd.

Reply:
i see them now. thnx!